build.user.properties.template

```diff
@@ -0,0 +1,21 @@
+nativity.dir=C:/Users/sync/Dev/streem/liferay-nativity
```

Comment:
These paths are used in the liferay solution so if the paths are relative, they have to be relative to whatever the working directory is where the build is run. What we can do is have a script that generates a properties file for you, but that is not the scope of this PR.